### ISSUE TITLE
Revision of The Reinforcement Module II

### ISSUE THREAD

**Issue by oOKexOo** [ArmaAchilles/AresModAchillesExpansion#213]
### Tasks:
- [ ] Feature: Customized filling of vehicles (e.g. #17)
### Requirements:
- [ ] Feature: RscListNBox for Achilles_fnc_ShowChooseDialog

**Comment by CreepPork** [comment]
Why didn't you add to the first issue these points? But ok, it's fine.

**Comment by ArchangelM127** [comment]
My idea, for your perusal, hopefully now in the right place:

A possibly simpler idea for expanding on the reinforcement function would be to add one or two new fields in the Spawn Reinforcements dialog, so the user could specify two or three different (or the same) groups to spawn in the selected vehicle. Any soldiers that don't fit would be auto-deleted, as now. So the user could either spawn several small groups into the same truck, or several full squads to parachute out of a C-130.

**Comment by oOKexOo** [comment]
Yes, this is essentially, what this issue is all about. `RscListNBox` is the control used for the inventory attribute window. It's the thing you have a list of items and you can add more/remove them with +/-.

**Comment by CreepPork** [comment]
`RscListNBox` is a terrible name, these aren't the days of rock n' roll :D
